modules/hostap/src/supp_main.c

```diff
@@ -0,0 +1,546 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+#include <sys/fcntl.h>
+LOG_MODULE_REGISTER(wpa_supplicant, LOG_LEVEL_DBG);
+
+#if defined(CONFIG_WPA_SUPP_CRYPTO) && !defined(CONFIG_MBEDTLS_ENABLE_HEAP)
+#include <mbedtls/platform.h>
+#endif /* CONFIG_WPA_SUPP_CRYPTO */
+
+#include <zephyr/net/socket.h>
+#include <zephyr/net/socket_select.h>
+#include <zephyr/net/wifi_nm.h>
+
+#include "includes.h"
+#include "common.h"
+#include "eloop.h"
+#include "wpa_supplicant/config.h"
+#include "wpa_supplicant_i.h"
+
+#include "fst/fst.h"
+#include "includes.h"
+#include "p2p_supplicant.h"
+#include "driver_i.h"
+
+#include "supp_main.h"
+#include "supp_events.h"
+#include "wpa_cli_zephyr.h"
+
+#include "supp_api.h"
+
+K_SEM_DEFINE(z_wpas_ready_sem, 0, 1);
+#include <l2_packet/l2_packet.h>
+
+/* Should match with the driver name */
+#define DEFAULT_IFACE_NAME "wlan0"
```

Comment:
Should we make this configurable?

Reply:
This was also gone already, the correct interface is detected automatically.